pep-0705.rst

```diff
@@ -0,0 +1,259 @@
+PEP: 9999
+Title: TypedMapping: Type Hints for Mappings with a Fixed Set of Keys
+Author: Alice Purcell <alicederyn@gmail.com>
+Sponsor: Pablo Galindo <pablogsal@gmail.com>
+Discussions-To: https://mail.python.org/archives/list/python-dev@python.org/thread/2P26R4VH2ZCNNNOQCBZWEM4RNF35OXOW/
```

Comment:
The discussions-to thread should be a new thread specifically about the PEP. Since this is very much a type system feature, I would strongly encourage you to have a discussion on typing-sig first.

Reply:
And just to clarify, you can wait to create this thread until right before or right after this PR is merged—just make sure to update `Discussions-To` and add an entry to `Post-History` accordingly.